scratch/PCA/extract_data.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
Is this file being intentionally committed, in spite of `*.ipynb` being added to the `.gitignore`?